app/Models/Meeting.php

```diff
@@ -10,7 +10,7 @@
 class Meeting extends Model
 {
     use HasFactory, SoftDeletes;
-    protected $fillable = ['user_id', 'title', 'date', 'start_at', 'room_id', 'level_id', 'category_id'];
+    protected $fillable = ['user_id', 'title', 'date', 'start_at', 'room_id', 'level_id', 'category_id', 'id'];
```

Comment:
the `id` should not be included in the `fillable` field since we have that field auto incremented.

Reply:
I see, I'll remove the 'id'.